<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>934</width>
    <height>566</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QGridLayout" name="gridLayout_6">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="Screen" name="screen">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="sizeIncrement">
       <size>
        <width>2</width>
        <height>1</height>
       </size>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>934</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFichier">
    <property name="title">
     <string>Fichier</string>
    </property>
    <addaction name="actionOpenFile"/>
    <addaction name="actionQuitter"/>
   </widget>
   <widget class="QMenu" name="menuConfig">
    <property name="title">
     <string>Config</string>
    </property>
    <addaction name="actionMode_avanc"/>
    <addaction name="separator"/>
    <addaction name="actionIgnore_read_write_errors"/>
   </widget>
   <addaction name="menuFichier"/>
   <addaction name="menuConfig"/>
  </widget>
  <widget class="QToolBar" name="controlBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>true</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionPlay"/>
   <addaction name="actionPause"/>
   <addaction name="actionStep_into"/>
   <addaction name="actionStep_over"/>
   <addaction name="actionReload"/>
  </widget>
  <widget class="QDockWidget" name="EmulatorLog">
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="allowedAreas">
    <set>Qt::AllDockWidgetAreas</set>
   </property>
   <property name="windowTitle">
    <string>Emulator log</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <property name="contextMenuPolicy">
     <enum>Qt::PreventContextMenu</enum>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <property name="sizeConstraint">
      <enum>QLayout::SetMaximumSize</enum>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <property name="spacing">
      <number>0</number>
     </property>
     <item row="0" column="0">
      <widget class="QPlainTextEdit" name="emulatorLogText">
       <property name="backgroundVisible">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionOuvrir_un_g1a">
   <property name="text">
    <string>Ouvrir un .g1a</string>
   </property>
  </action>
  <action name="actionQuitter">
   <property name="text">
    <string>Quitter</string>
   </property>
  </action>
  <action name="actionMode_avanc">
   <property name="text">
    <string>Mode avanc√©</string>
   </property>
  </action>
  <action name="actionPause">
   <property name="icon">
    <iconset resource="ose.qrc">
     <normaloff>:/icones/ico/pause</normaloff>:/icones/ico/pause</iconset>
   </property>
   <property name="text">
    <string>Pause</string>
   </property>
   <property name="shortcut">
    <string>F9</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionPlay">
   <property name="icon">
    <iconset resource="ose.qrc">
     <normaloff>:/icones/ico/play</normaloff>:/icones/ico/play</iconset>
   </property>
   <property name="text">
    <string>Play</string>
   </property>
   <property name="shortcut">
    <string>F8</string>
   </property>
  </action>
  <action name="actionReload">
   <property name="icon">
    <iconset resource="ose.qrc">
     <normaloff>:/icones/ico/reload</normaloff>:/icones/ico/reload</iconset>
   </property>
   <property name="text">
    <string>Reload</string>
   </property>
  </action>
  <action name="actionStep_into">
   <property name="icon">
    <iconset resource="ose.qrc">
     <normaloff>:/icones/ico/stepinto</normaloff>:/icones/ico/stepinto</iconset>
   </property>
   <property name="text">
    <string>Step into</string>
   </property>
   <property name="shortcut">
    <string>F10</string>
   </property>
  </action>
  <action name="actionStep_over">
   <property name="icon">
    <iconset resource="ose.qrc">
     <normaloff>:/icones/ico/stepover</normaloff>:/icones/ico/stepover</iconset>
   </property>
   <property name="text">
    <string>Step over</string>
   </property>
  </action>
  <action name="actionIgnore_read_write_errors">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Ignore read/write errors</string>
   </property>
  </action>
  <action name="actionOpenFile">
   <property name="text">
    <string>Open a binary file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Screen</class>
   <extends>QLabel</extends>
   <header>screen.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="ose.qrc"/>
 </resources>
 <connections/>
</ui>
